src/utils/simple_logger.cpp

```diff
@@ -40,7 +40,6 @@ namespace dsn {
 namespace tools {
 
 DSN_DEFINE_bool(tools.simple_logger, fast_flush, false, "whether to flush immediately");
-
 DSN_DEFINE_bool(tools.simple_logger,
                 short_header,
```

Comment:
Add another `short_header` for `tools.screen_logger` ?

Reply:
It seems `screen_logger(const char *log_dir)` is only used in one test, and the parameter `log_dir` is useless, so I removed the c'stor and updated the only caller.

ps: Two sections with one same key is not implemented yet.

Reply:
OK, that means `[tools.screen_logger].short_header` is not used anywhere.